### ISSUE TITLE
Restore commits from bugfix-#103

### ISSUE THREAD

**Issue by ptatian** [NeighborhoodInfoDC/ACS#110]

**Comment by ptatian** [comment]
@rpitingolo OK, I think I have a plan for fixing this using the [cherry-pick](https://git-scm.com/docs/git-cherry-pick) command. 

I started the process to see if it would work. First step was to revert back to how things were at the end of bugfix-#103. From the Git command prompt in the local ACS repo.

```
> git checkout master
> git pull
> git checkout -b bugfix-#110 f06dc90
```

This rolled back to where master was after bugfix-#103 and created a new bugfix-#110 branch. 

Next we want to [sequentially reapply the commits since then](https://github.com/NeighborhoodInfoDC/ACS/commits/master). Back in the command prompt, 

```
> git cherry-pick 68818e0 
```

Cherry-pick creates a new commit. I would then go to the Desktop app and examine the changes after each cherry-pick command. There may be conflicts or changes from the previous commit that you need to edit manually before you go to the next. 

```
> git cherry-pick b3f4f1f 
```

It's a little tedious, but it looks like we will be able to reapply all of the changes, using bugfix-#103 as the base. 

I did the first two commits (68818e0 and b3f4f1f) already and pushed those changes. So if you fetch and check out the bugfix-#110 branch you can continue with the next commit d260305 on [through the list](https://github.com/NeighborhoodInfoDC/ACS/commits/master) up to the last commit on June 8. Note that you cannot cherry-pick **merge** commits (the ones with Verified next to them in the commit list). If you try, you will just get an error. So, just skip the merge commits. But, you might want to test the programs when you get to a merge commit -- since they should run properly at that point. 

If we are satisfied with all of the fixes, we can do a pull request, merge back to master, and resubmit everything. 

Let me know if you have any questions. 

Peter

cc: @lhendey

**Comment by rpitingolo** [comment]
@ptatian I checked out the branch and tried to do git cherry-pick 73ffc5b but am getting this error: 

error: Committing is not possible because you have unmerged files.
hint: Fix them up in the work tree, and then use 'git add/rm <file>'
hint: as appropriate to mark resolution and make a commit.
fatal: Exiting because of an unresolved conflict.
U       Macros/ACS_summary_geo_source_tr_vars.sas

Based on some googling it looks like I should use `git add Macros/ACS_summary_geo_source_tr_vars.sas` to resolve. Does that seem right to you?

**Comment by rpitingolo** [comment]
@ptatian I think I skipped over d260305 so I aborted the original cherry-pick and tried  to do d260305 but still get the same error.

**Comment by ptatian** [comment]
@rpitingolo Yes, I was able to reproduce that and it's because there are conflicts. For d2603306 it should be Macros/ACS_summary_geo.sas that has the conflicts. You need to edit that file to remove the conflicts and then you can commit those changes. So, this is where it will get tricky because you need to make sure you keep all the right changes.

**Comment by rpitingolo** [comment]
@ptatian ok, sorry if this is a dumb question, but how do I actually see what the conflicts are?

**Comment by ptatian** [comment]
@rpitingolo You look for the `<<<<<<<` and `>>>>>>>` marks in the file. 

Most likely you are going to want to just combine all the changes because, as I understand it, the problem was earlier changes got eliminated in later commits.

**Comment by rpitingolo** [comment]
@ptatian that's what I figured, but when I open the program in sas it doesn't have those marks in the file. I've run git status to double check that I'm looking at the right file, just can't find any apparent conflicts. If you have bugfix-#110 checked out are you able to see conflicts in Macros/ACS_summary_geo.sas ?

**Comment by ptatian** [comment]
Yes, I can see them. Are you sure you are looking at the right file? 

```
       NumFamilies_: NumHshlds_:

       PopUnder5Years_: PopUnder18Years_:
       Pop18_34Years_: Pop35_64Years_:
       Pop65andOverYears_:

       Pop25andOverYears_: Pop25andOverWoutHS_: Pop25andOverWHS_:
       Pop25andOverWSC_: Pop25andOverWCollege_:
<<<<<<< HEAD
               
=======

       FamIncomeLT75k_: FamIncomeGT200k_:
	
	   
       NumRenterCostBurden: NumRentSevereCostBurden: NumOwnerCostBurden: NumOwnSevereCostBurden:
       RentCostBurdenDenom: OwnerCostBurdenDenom:

       NumRentCst: NumOwnCstBurden:
      
>>>>>>> d260305... Update ACS_SF and ACS_summary_geo to include chanegs
       ;
           
    %let moe_vars =
           mTotPop_&_years. mPopUnder5Years_&_years. 
       mPopUnder18Years_&_years. mPop18_34Years_&_years.
       mPop35_64Years_&_years. mPop65andOverYears_&_years.
       mPop25andOverYears_&_years.
```

**Comment by rpitingolo** [comment]
@ptatian I'm looking at D:\DCData\Libraries\ACS\Macros\ACS_summary_geo.sas while checked out in bugfix-#110 and don't see the marks or code in-between. 

I've tried git fetch, checked git status and it still shows Macros\ACS_summary_geo.sas as the file with conflicts. I just can't get them to show up so I can edit. I don't see the conflicts on the web view either: https://github.com/NeighborhoodInfoDC/ACS/blob/bugfix-%23110/Macros/ACS_summary_geo.sas

**Comment by ptatian** [comment]
@rpitingolo What do you see when you do `git status`

**Comment by rpitingolo** [comment]
@ptatian sorry, I'm back on track. I think I was using the git shell syntax incorrectly at first but have it working now.

**Comment by lhendey** [comment]
@rpitingolo  = we can close this issue right?
